src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/SchoolYearValidation.feature

```diff
@@ -0,0 +1,273 @@
+@SchoolYear
+Feature: School Year Reference Validation
+  Reference validation on School Years
+
+  """
+  @StudentEducationOrganizationAssociation Hook that will create the following data for the required tests
+  createStudent(studentUniqueId)
+  createSchool(schoolId)
+  createCourse(courseCode, schoolId)
+  createSession(schoolId, schoolYear, sessionName)
+  createCourseOffering(localCourseCode, courseCode, schoolId, schoolYear, sessionName)
+  createSection(localCourseCode, schoolId, schoolYear, sessionName, sectionIdentifier)
+  """
+
+  Background:
+    Given a created student with "<studentUniqueId>"
```

Comment:
This will not be enough information to create a student. If the other values don't matter in this situation (like `firstName`), then this does seem fine. On the back end we can provide the additional information. However, whoever completes this will need to give a proper `studentUniqueId` value here. On line 37 you have 604822, therefore it would make sense to put 604822 above. When you do that, the reader won't know what the number means. Therefore, perhaps it would be good to write `Given a student with unique Id "604822"`.

Similarly, below you might change to `Given a school with Id 255901001`.

Or... you could use the exact same syntax as Esteban in [TotalCountQueryString.feature.disabled](https://github.com/Ed-Fi-Alliance-OSS/Data-Management-Service/blob/main/src/services/EdFi.DataManagementService.Api.Tests.E2E/Features/TotalCountQueryString.feature.disabled). If you use the same syntax, then we only need one back-end hook that will apply in both cases.